Doc/library/logging.rst

```diff
@@ -1034,12 +1034,12 @@ interchangeably.
    to :class:`LoggerAdapter`.  These methods delegate to the underlying logger.
 
 .. versionchanged:: 3.6
-   Attribute :attr:`manager` and method :meth:`_log` were added, which
+   Attribute :attr:`!manager` and method :meth:`!_log` were added, which
    delegate to the underlying logger and allow adapters to be nested.
 
 .. versionchanged:: 3.13
-   Remove the undocumented ``warn()`` method which was an alias to the
-   ``warning()`` method.
+   Remove the undocumented :meth:`!warn`` method which was an alias to the
+   :meth:`!warning` method.
 
 .. versionchanged:: 3.13
    The *merge_extra* argument was added.
```

Comment:
This entire section (starting "In addition to the above ...") should be indented by 3 spaces to fall under ``.. class:: LoggerAdapter``

Reply:
Indeed; good observation!

Reply:
There was another paragraph needing indentation, I've pushed a commit to fix (hope you don't mind!)

A